### DISCUSSION TITLE
新项目不再推荐使用 Reduck，是不是意味着未来不再内置状态管理了？

### DISCUSSION THREAD

**Discussion by ibyxz** [web-infra-dev/modern.js#6546] [General]
看文档介绍中，提示“新项目不再推荐使用 Reduck，可以使用社区中的状态管理工具，例如 [Jotai]、[zustand]、[valtio]等。”，现在比较纠结用哪个状态管理库，如果未来有计划更换掉reduck，有没有具体方向可以建议的。

**Comment by zllkjc** [comment]
是的，我们推荐使用社区中更先进的状态管理库~ zustand、Jotai 等都是非常概念简洁且被社区广泛使用的库。
